src/swap/types.ts

```diff
@@ -217,24 +234,36 @@ export type SwapLoadingState = {
  * Note: exported as public Type
  */
 export type SwapMessageReact = {
-  className?: string; // Optional className override for top div element.
+  /** Optional className override for top div element */
+  className?: string;
 };
 
 /**
  * Note: exported as public Type
  */
 export type SwapQuote = {
-  amountReference: string; // The reference amount for the quote
-  from: Token; // The source token for the swap
-  fromAmount: string; // The amount of the source token
-  fromAmountUSD: string; // The USD value of the source token
-  hasHighPriceImpact: boolean; // Whether the price impact is high
-  priceImpact: string; // The price impact of the swap
-  slippage: string; // The slippage of the swap
-  to: Token; // The destination token for the swap
-  toAmount: string; // The amount of the destination token
-  toAmountUSD: string; // The USD value of the destination token
-  warning?: QuoteWarning; // The warning associated with the quote
+  /** The reference amount for the quote */
```

Comment:
non-blocking: i don't know what a reference amount is, and this comment doesn't help me understand